kuma/static/js/analytics.js

```diff
@@ -1,8 +1,20 @@
 (function(win, doc, $) {
     'use strict';
 
+    // prettier-ignore
```

Comment:
We could make it a pre-commit hook. But that is a fair comment. Let me revert those changes and simply add the `postMessage` additions.